light_node/src/configuration.rs

```diff
@@ -842,24 +851,42 @@ impl Environment {
                     columns: ContextActionsTableInitializer {},
                     threads: db_context_actions_threads_count,
                 };
-                let store_context_actions = args
-                    .value_of("store-context-actions")
-                    .unwrap_or("true")
-                    .parse::<bool>()
-                    .expect("Provided value cannot be converted to bool");
                 let compute_context_action_tree_hashes = args
                     .value_of("compute-context-action-tree-hashes")
                     .unwrap_or("false")
                     .parse::<bool>()
                     .expect("Provided value cannot be converted to bool");
+
+                let mut backends = match args.values_of("actions-store-backend") {
+                    Some(v) => v.map(String::from).collect(),
+                    None => {
+                        vec!["rocksdb".to_string()]
+                    }
+                };
+                backends.dedup();
```

Comment:
@mat13mn 
according to docs, if understand it, this will work only if `backends` is sorted before,
so I guess, if backends will be: `["rocksdb", "none", "rocksdb"]`, it will not change anything,

maybe better is to `v.map(String::from).collect()` collect as HashSet or something like that